### DISCUSSION TITLE
Mapping page to multiple routes

### DISCUSSION THREAD

**Discussion by milenazuccarelli** [vercel/next.js#15514] [Help] [answered]
Hey guys,

I have a dynamic route which looks like this:
`/chapters/[slug].js`
But I would like the first chapter `/chapters/chapter-i` to also be the landing page/index for the website. Could anyone suggest how I should go about doing this? (I'm not using a custom server). 

Thanks!

**Comment by TommySorensen** [comment, marked as answer]
Optional Catch all routes is what you are looking for 😉
https://nextjs.org/docs/routing/dynamic-routes#optional-catch-all-routes

**Reply by milenazuccarelli** [reply]
Thank you! Maybe I'm missing something though, do you know how I can map `pages/index.js` to `pages/chapters/chapter-i` ?

**Reply by TommySorensen** [reply]
Like mentioned in the documentation just create a Catch all route under the pages folder. This page will be hit for all urls.
```
pages/[[...slug]].js
```

**Comment by alanqchen** [comment]
I would try looking at [rewrites](https://nextjs.org/docs/api-reference/next.config.js/rewrites) or [redirects](https://nextjs.org/docs/api-reference/next.config.js/redirects), depending on if you want to original URL to change or not.

For example, you could rewrite `/` to `/chapters/chapter-i`:

```js
module.exports = {
  async rewrites() {
    return [
      {
        source: '/',
        destination: '/chapters/chapter-i',
      },
    ]
  },
}
```
